docs/run_validator.md

```diff
@@ -225,157 +178,235 @@ You should see this:
 
 ### Prepare the staking address
 
-We included 1 million Ethereum addresses, over 8000 Cosmos addresses and all of `euler-4` validators addresses into  genesis, so there's a huge chance that you alredy have some EUL tokens. Here are 3 ways to check this:
+We included 1 million Ethereum addresses, over 8000 Cosmos addresses and all of `euler-4` validators addresses into the genesis file. This means that there's a huge chance that you alredy have some EUL tokens. Here are 3 ways to check this:
 
 If you already have a cyberd address with EUL and know the seed phrase or your private key, just restore it into your local keystore:
 
 ```bash
-docker exec -ti euler-5 cyberdcli keys add <your_key_name> --recover
-docker exec euler-5 cyberdcli keys show <your_key_name>
+cyberdcli keys add <your_key_name> --recover
+cyberdcli keys show <your_key_name>
 ```
 
-If you have an Ethereum address that had ~0.2Eth or more at block 8080808 (on the ETH network), you can import your Ethereum private key. To do this, please check out our Ethereum [gift tool](qhttps://github.com/cybercongress/launch-kit/tree/0.1.0/ethereum_gift_tool)
+If you have an Ethereum address that had ~0.2Eth or more at block 8080808 (on the ETH network), you probably recieved a gift and may import your Ethereum private key. To check your gift balance, paste your Ethereum address on [cyber.page](https://cyber.page).
 
-> Please do not import high value Ethereum accounts. This is not safe! cyberd software is a new and has not been battle tested yet.
+> Please do not import high value Ethereum accounts. This is not safe! cyberd software is new and has not been audited yet.
 
 ```bash
-docker exec -ti euler-5 cyberdcli keys add import_private <your_key_name>
-docker exec euler-5 cyberdcli keys show <your_key_name>
+cyberdcli keys add private <your_key_name>
+cyberdcli keys show <your_key_name>
 ```
 
 If you want to create a new acccount, use the command below:
-(You should send coins to that address to bound them later during the submitting of the validator)
+(You should send coins to that address to bound them later during the launch of the validator)
 
 ```bash
-docker exec -ti euler-5 cyberdcli keys add <your_key_name>
-docker exec euler-5 cyberdcli keys show <your_key_name>
+cyberdcli keys add <your_key_name>
+cyberdcli keys show <your_key_name>
 ```
 
-You could use your ledger device with the Cosmos app installed on it to sign and store cyber addresses: [guide here](https://github.com/cybercongress/cyberd/blob/0.1.5/docs/cyberd_Ledger_guide.md).
+You could use your Ledger device, with the Cosmos app installed on it to sign and store cyber addresses: [guide here](https://github.com/cybercongress/cyberd/blob/0.1.5/docs/cyberd_Ledger_guide.md).
 In common case use the --ledger flag, with your commands:
 
 ```bash
-docker exec -ti euler-5 cyberdcli keys add <your_key_name> --ledger
+cyberdcli keys add <your_key_name> --ledger
```

Comment:
add notes to use with keyring and local file, not only with ledger @mrlp4

Reply:
done